### PR TITLE
add parameters for dupe detection service in SN config file

### PR DESCRIPTION
**Pull request by georgkcode0**
https://freedcamp.com/view/2956207/tasks/41229447/panel/task/41229472

### CODE REVIEW

**Comment by pjenys0** [comment]
Could you put freecamp task linked to this PR in description? also describe shortly the changes of this PR?

**Comment by georgkcode0** [comment]
> Could you put freecamp task linked to this PR in description? also describe shortly the changes of this PR?

You can refer to https://freedcamp.com/view/2956207/tasks/41229447/panel/task/41229472

**Review by pjenys0 — APPROVED** [review]
LGTM!

**Review by danileng20 — APPROVED** [review]
LGTM!

File: probe/pkg/dupedetection/config.go [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	// DefaultInputDir that dd-service monitors for the new file to generate fingerprints
+	DefaultInputDir = "input"
+	// DefaultOutputDir that dd-service uses to put new fingerprints
+	DefaultOutputDir = "output"
+	// DefaultDataFile is path to the SQLite database used by dd-service
+	DefaultDataFile = "dupe_detection_image_fingerprint_database.sqlite"
+)
+
+// Config contains settings of the dupe detection service
+type Config struct {
```
Comment (danileng20) [inline probe/pkg/dupedetection/config.go:15]: I think this config should be used in the tensor structure in tensor.go. And it should be passed through the supernode app when it starts.

**Pull request merged**